src/Extensions/CommentsExtension.php

```diff
@@ -463,7 +463,7 @@ public function CommentsForm()
         $enabled = $this->getCommentsEnabled();
         if ($enabled && $this->owner->getCommentsOption('include_js')) {
             Requirements::javascript('//code.jquery.com/jquery-3.3.1.min.js');
-            Requirements::javascript('silverstripe/comments:thirdparty/jquery-validate/jquery.validate.min.js');
+            Requirements::javascript('silverstripe/comments:thirdparty/jquery-validation/jquery.validate.min.js');
```

Comment:
Even though third party libraries aren't part of our public API, this path name change might cause problems for anyone relying on those files to exist and would introduce unnecessary risk for a patch release. Can you rename it back to what it was?

Reply:
huh, yeah. Strange the library (as per filename) is jquery.validate - but the folder name came direct from release distribution; 'jquery-validation'. Weird.

Sure I'll put it back.

Reply:
I have resolved this comment, but I'll let you resolve this conversation :)